hw/top_earlgrey/ip/ast/ast_wrapper.core

```diff
@@ -9,6 +9,8 @@ filesets:
     depend:
       - lowrisc:systems:ast_wrapper_pkg
       - lowrisc:systems:ast
+      - lowrisc:top_earlgrey:clkmgr
+      - lowrisc:systems:rstmgr_pkg
       - lowrisc:ip:entropy_src
```

Comment:
`lowrisc:ip:entropy_src_pkg` should be sufficient.

Reply:
ah one thing i should point out... @vogelpi until we resolve the right way to handle via your top level PR, it should be enough to do `lowrisc:ip:rstmgr_pkg" here, and the flags should re-direct.

Reply:
Yep indeed. I've switched to `lowrisc:ip:entropy_src_pkg` and `lowrisc:ip:rstmgr_pkg`.